### ISSUE TITLE
Installation guide

### ISSUE THREAD

**Issue by v0d1ch** [JackKiefer/herms#5]
It would be helpful to describe installation process

**Comment by gallais** [comment]
Yeah, a cabal file would be welcome especially now that there are external dependencies (Brick).

**Comment by CharlesHD** [comment]
Indeed a cabal file would be great. I personnaly like using [stack](https://docs.haskellstack.org/en/stable/README/) for ease of use.

**Comment by JackKiefer** [comment]
Yes! Let's get stack working :)

**Comment by JackKiefer** [comment]
Update! Everyone! 
There is now a cabal file and a stack.yaml. Wow! Cool stuff!
Though, I have no idea what I'm doing. You can use them for things? 
This is fun.

**Comment by Jorissss** [comment]
Hi, the cabal installation described in the guide doesn't seem to work for me (ghc 8.2.1).
I get the follow output:

Resolving dependencies...
Notice: installing into a sandbox located at
/home/joris/forks/herms/.cabal-sandbox
Configuring StateVar-1.1.0.4...
Configuring dlist-0.8.0.3...
Configuring microlens-0.4.8.1...
Configuring blaze-builder-0.4.0.2...
Configuring parallel-3.2.1.1...
Configuring data-clist-0.1.2.0...
Configuring text-zipper-0.10.1...
Configuring semigroups-0.18.3...
Building StateVar-1.1.0.4...
Building dlist-0.8.0.3...
Building microlens-0.4.8.1...
Building parallel-3.2.1.1...
Building blaze-builder-0.4.0.2...
Building data-clist-0.1.2.0...
Building text-zipper-0.10.1...
Building semigroups-0.18.3...
Failed to install StateVar-1.1.0.4
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/StateVar-1.1.0.4-H2MMzWhEyt5Bk7Ng0WRNLx.log ):
cabal: Entering directory '/tmp/cabal-tmp-31447/StateVar-1.1.0.4'
Configuring StateVar-1.1.0.4...
Preprocessing library for StateVar-1.1.0.4..
Building library for StateVar-1.1.0.4..
[1 of 1] Compiling Data.StateVar    ( src/Data/StateVar.hs, dist/dist-sandbox-ebcec679/build/Data/StateVar.o )

src/Data/StateVar.hs:66:8: error:
    Could not find module ‘Prelude’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
66 | module Data.StateVar
   |        ^^^^^^^^^^^^^

src/Data/StateVar.hs:80:1: error:
    Could not find module ‘Control.Concurrent.STM’
    There are files missing in the ‘stm-2.4.4.1’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
80 | import Control.Concurrent.STM
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/StateVar.hs:81:1: error:
    Could not find module ‘Control.Monad.IO.Class’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
81 | import Control.Monad.IO.Class
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/StateVar.hs:82:1: error:
    Could not find module ‘Data.IORef’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
82 | import Data.IORef
   | ^^^^^^^^^^^^^^^^^

src/Data/StateVar.hs:83:1: error:
    Could not find module ‘Data.Typeable’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
83 | import Data.Typeable
   | ^^^^^^^^^^^^^^^^^^^^

src/Data/StateVar.hs:84:1: error:
    Could not find module ‘Foreign.Ptr’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
84 | import Foreign.Ptr
   | ^^^^^^^^^^^^^^^^^^

src/Data/StateVar.hs:85:1: error:
    Could not find module ‘Foreign.Storable’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
85 | import Foreign.Storable
   | ^^^^^^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-31447/StateVar-1.1.0.4'
Configuring transformers-compat-0.5.1.4...
Failed to install dlist-0.8.0.3
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/dlist-0.8.0.3-8XCsMwviZunHg7vFDzvN5q.log ):
cabal: Entering directory '/tmp/cabal-tmp-31450/dlist-0.8.0.3'
Configuring dlist-0.8.0.3...
Preprocessing library for dlist-0.8.0.3..
Building library for dlist-0.8.0.3..
[1 of 1] Compiling Data.DList       ( Data/DList.hs, dist/dist-sandbox-ebcec679/build/Data/DList.o )

Data/DList.hs:60:1: error:
    Could not find module ‘Prelude’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
60 | import Prelude hiding (concat, foldr, map, head, tail, replicate)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:61:1: error:
    Could not find module ‘Data.List’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
61 | import qualified Data.List as List
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:62:1: error:
    Could not find module ‘Control.DeepSeq’
    There are files missing in the ‘deepseq-1.4.3.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
62 | import Control.DeepSeq (NFData(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:63:1: error:
    Could not find module ‘Control.Monad’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
63 | import Control.Monad as M
   | ^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:64:1: error:
    Could not find module ‘Data.Function’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
64 | import Data.Function (on)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:65:1: error:
    Could not find module ‘Data.String’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
65 | import Data.String (IsString(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:67:1: error:
    Could not find module ‘Data.Foldable’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
67 | import qualified Data.Foldable as F
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:76:1: error:
    Could not find module ‘Data.Semigroup’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
76 | import Data.Semigroup (Semigroup(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:81:1: error:
    Could not find module ‘Text.Read’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
81 | import Text.Read (Lexeme(Ident), lexP, parens, prec, readPrec, readListPrec,
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...

Data/DList.hs:85:1: error:
    Could not find module ‘GHC.Exts’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
85 | import GHC.Exts (IsList)
   | ^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:87:1: error:
    Could not find module ‘GHC.Exts’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
87 | import qualified GHC.Exts
   | ^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:92:1: error:
    Could not find module ‘Control.Applicative’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
92 | import Control.Applicative(Alternative, (<|>))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Data/DList.hs:93:1: error:
    Could not find module ‘Control.Applicative’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
93 | import qualified Control.Applicative (empty)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-31450/dlist-0.8.0.3'
Failed to install text-zipper-0.10.1
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/text-zipper-0.10.1-5anw58L5L9nGBx6ICflo2w.log ):
cabal: Entering directory '/tmp/cabal-tmp-31454/text-zipper-0.10.1'
Configuring text-zipper-0.10.1...
Preprocessing library for text-zipper-0.10.1..
Building library for text-zipper-0.10.1..
[1 of 4] Compiling Data.Text.Zipper.Vector ( src/Data/Text/Zipper/Vector.hs, dist/dist-sandbox-ebcec679/build/Data/Text/Zipper/Vector.o )

src/Data/Text/Zipper/Vector.hs:1:8: error:
    Could not find module ‘Prelude’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
  |
1 | module Data.Text.Zipper.Vector
  |        ^^^^^^^^^^^^^^^^^^^^^^^

src/Data/Text/Zipper/Vector.hs:6:1: error:
    Could not find module ‘Data.Vector’
    There are files missing in the ‘vector-0.12.0.1’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
  |
6 | import qualified Data.Vector as V
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-31454/text-zipper-0.10.1'
Failed to install microlens-0.4.8.1
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/microlens-0.4.8.1-HUhGDwljLwYJLac51xSk3I.log ):
Failed to install parallel-3.2.1.1
Failed to install data-clist-0.1.2.0
cabal: Entering directory '/tmp/cabal-tmp-31451/microlens-0.4.8.1'
Configuring microlens-0.4.8.1...
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/parallel-3.2.1.1-BDA8bigzIyf7HfOpmZ9w2K.log ):
Preprocessing library for microlens-0.4.8.1..
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/data-clist-0.1.2.0-GRWlwOEuIsQA9GA7i2yMwp.log ):
Building library for microlens-0.4.8.1..
[1 of 4] Compiling Lens.Micro.Type  ( src/Lens/Micro/Type.hs, dist/dist-sandbox-ebcec679/build/Lens/Micro/Type.o )
cabal: Entering directory '/tmp/cabal-tmp-31452/parallel-3.2.1.1'

cabal: Entering directory '/tmp/cabal-tmp-31449/data-clist-0.1.2.0'
Configuring parallel-3.2.1.1...
src/Lens/Micro/Type.hs:13:8: error:
Configuring data-clist-0.1.2.0...
Preprocessing library for parallel-3.2.1.1..
    Could not find module ‘Prelude’
Preprocessing library for data-clist-0.1.2.0..
Building library for parallel-3.2.1.1..
    There are files missing in the ‘base-4.10.0.0’ package,
Building library for data-clist-0.1.2.0..
[1 of 3] Compiling Control.Parallel ( Control/Parallel.hs, dist/dist-sandbox-ebcec679/build/Control/Parallel.o )
    try running 'ghc-pkg check'.
[1 of 1] Compiling Data.CircularList ( src/Data/CircularList.hs, dist/dist-sandbox-ebcec679/build/Data/CircularList.o )

    Use -v to see a list of the files searched for.

Control/Parallel.hs:17:8: error:
   |
src/Data/CircularList.hs:72:1: error:
    Could not find module ‘Prelude’
13 | module Lens.Micro.Type
    Could not find module ‘Control.Applicative’
    There are files missing in the ‘base-4.10.0.0’ package,
   |        ^^^^^^^^^^^^^^^
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.

    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
src/Lens/Micro/Type.hs:25:1: error:
    Use -v to see a list of the files searched for.
   |
    Could not find module ‘Control.Applicative’
   |
17 | module Control.Parallel (
    There are files missing in the ‘base-4.10.0.0’ package,
72 | import Control.Applicative hiding (empty)
   |        ^^^^^^^^^^^^^^^^
    try running 'ghc-pkg check'.
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

    Use -v to see a list of the files searched for.

Control/Parallel.hs:22:1: error:
   |
src/Data/CircularList.hs:73:1: error:
    Could not find module ‘GHC.Conc’
25 | import Control.Applicative
    Could not find module ‘Prelude’
    There are files missing in the ‘base-4.10.0.0’ package,
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.

    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
src/Lens/Micro/Type.hs:26:1: error:
    Use -v to see a list of the files searched for.
   |
    Could not find module ‘Data.Functor.Identity’
   |
22 | import qualified GHC.Conc       ( par, pseq )
    There are files missing in the ‘base-4.10.0.0’ package,
73 | import Prelude
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    try running 'ghc-pkg check'.
   | ^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-31452/parallel-3.2.1.1'
    Use -v to see a list of the files searched for.

   |
src/Data/CircularList.hs:74:1: error:
26 | import Data.Functor.Identity
    Could not find module ‘Data.List’
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    There are files missing in the ‘base-4.10.0.0’ package,
cabal: Leaving directory '/tmp/cabal-tmp-31451/microlens-0.4.8.1'
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
74 | import Data.List(find,unfoldr,foldl')
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/CircularList.hs:75:1: error:
    Could not find module ‘Control.DeepSeq’
    There are files missing in the ‘deepseq-1.4.3.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
75 | import Control.DeepSeq(NFData(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/CircularList.hs:76:1: error:
    Could not find module ‘Control.Monad’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
76 | import Control.Monad(join)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/CircularList.hs:77:1: error:
    Could not find module ‘Data.Traversable’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
77 | import qualified Data.Traversable as T
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/CircularList.hs:78:1: error:
    Could not find module ‘Data.Foldable’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
78 | import qualified Data.Foldable as F
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/CircularList.hs:80:1: error:
    Could not find module ‘Test.QuickCheck.Arbitrary’
    There are files missing in the ‘QuickCheck-2.10.0.1’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
80 | import Test.QuickCheck.Arbitrary
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/CircularList.hs:81:1: error:
    Could not find module ‘Test.QuickCheck.Gen’
    There are files missing in the ‘QuickCheck-2.10.0.1’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
81 | import Test.QuickCheck.Gen
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-31449/data-clist-0.1.2.0'
Failed to install semigroups-0.18.3
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/semigroups-0.18.3-9GRyDnRvoqO5HfdYR3Qz4o.log ):
cabal: Entering directory '/tmp/cabal-tmp-31453/semigroups-0.18.3'
Configuring semigroups-0.18.3...
Preprocessing library for semigroups-0.18.3..
Building library for semigroups-0.18.3..
[1 of 1] Compiling Data.Semigroup.Generic ( src/Data/Semigroup/Generic.hs, dist/dist-sandbox-ebcec679/build/Data/Semigroup/Generic.o )

src/Data/Semigroup/Generic.hs:23:8: error:
    Could not find module ‘Prelude’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
23 | module Data.Semigroup.Generic
   |        ^^^^^^^^^^^^^^^^^^^^^^

src/Data/Semigroup/Generic.hs:30:1: error:
    Could not find module ‘Data.Semigroup’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
30 | import Data.Semigroup
   | ^^^^^^^^^^^^^^^^^^^^^

src/Data/Semigroup/Generic.hs:31:1: error:
    Could not find module ‘GHC.Generics’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
31 | import GHC.Generics
   | ^^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-31453/semigroups-0.18.3'
Failed to install blaze-builder-0.4.0.2
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/blaze-builder-0.4.0.2-9jN75sESh05JfWRLWr51Wl.log ):
cabal: Entering directory '/tmp/cabal-tmp-31448/blaze-builder-0.4.0.2'
Configuring blaze-builder-0.4.0.2...
Preprocessing library for blaze-builder-0.4.0.2..
Building library for blaze-builder-0.4.0.2..
[ 1 of 10] Compiling Blaze.ByteString.Builder.Internal.Write ( Blaze/ByteString/Builder/Internal/Write.hs, dist/dist-sandbox-ebcec679/build/Blaze/ByteString/Builder/Internal/Write.o )

Blaze/ByteString/Builder/Internal/Write.hs:18:8: error:
    Could not find module ‘Prelude’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
18 | module Blaze.ByteString.Builder.Internal.Write (
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Blaze/ByteString/Builder/Internal/Write.hs:50:1: error:
    Could not find module ‘Foreign’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
50 | import Foreign
   | ^^^^^^^^^^^^^^

Blaze/ByteString/Builder/Internal/Write.hs:52:1: error:
    Could not find module ‘Control.Monad’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
52 | import Control.Monad
   | ^^^^^^^^^^^^^^^^^^^^

Blaze/ByteString/Builder/Internal/Write.hs:54:1: error:
    Could not find module ‘Data.ByteString.Builder.Internal’
    There are files missing in the ‘bytestring-0.10.8.2’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
54 | import Data.ByteString.Builder.Internal
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-31448/blaze-builder-0.4.0.2'
Building transformers-compat-0.5.1.4...
Failed to install transformers-compat-0.5.1.4
Build log ( /home/joris/forks/herms/.cabal-sandbox/logs/ghc-8.2.1/transformers-compat-0.5.1.4-HHfl4KyHCH71PMx2CRgdve.log ):
cabal: Entering directory '/tmp/cabal-tmp-31447/transformers-compat-0.5.1.4'
Configuring transformers-compat-0.5.1.4...
Preprocessing library for transformers-compat-0.5.1.4..
Building library for transformers-compat-0.5.1.4..
[1 of 2] Compiling Control.Monad.Trans.Instances ( src/Control/Monad/Trans/Instances.hs, dist/dist-sandbox-ebcec679/build/Control/Monad/Trans/Instances.o )

src/Control/Monad/Trans/Instances.hs:39:8: error:
    Could not find module ‘Prelude’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
39 | module Control.Monad.Trans.Instances () where
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:49:1: error:
    Could not find module ‘Control.Applicative.Backwards’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
49 | import           Control.Applicative.Backwards (Backwards(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:50:1: error:
    Could not find module ‘Control.Applicative.Lift’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
50 | import           Control.Applicative.Lift (Lift(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:51:1: error:
    Could not find module ‘Control.Monad.IO.Class’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
51 | import           Control.Monad.IO.Class (MonadIO)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:52:1: error:
    Could not find module ‘Control.Monad.Trans.Class’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
52 | import           Control.Monad.Trans.Class (MonadTrans)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:53:1: error:
    Could not find module ‘Control.Monad.Trans.Cont’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
53 | import           Control.Monad.Trans.Cont (ContT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:54:1: error:
    Could not find module ‘Control.Monad.Trans.Error’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
54 | import           Control.Monad.Trans.Error (ErrorT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:55:1: error:
    Could not find module ‘Control.Monad.Trans.Except’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
55 | import           Control.Monad.Trans.Except ()
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:56:1: error:
    Could not find module ‘Control.Monad.Trans.Identity’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
56 | import           Control.Monad.Trans.Identity (IdentityT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:57:1: error:
    Could not find module ‘Control.Monad.Trans.List’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
57 | import           Control.Monad.Trans.List (ListT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:58:1: error:
    Could not find module ‘Control.Monad.Trans.Maybe’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
58 | import           Control.Monad.Trans.Maybe (MaybeT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:59:1: error:
    Could not find module ‘Control.Monad.Trans.RWS.Lazy’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
59 | import qualified Control.Monad.Trans.RWS.Lazy as Lazy (RWST(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:60:1: error:
    Could not find module ‘Control.Monad.Trans.RWS.Strict’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
60 | import qualified Control.Monad.Trans.RWS.Strict as Strict (RWST(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:61:1: error:
    Could not find module ‘Control.Monad.Trans.Reader’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
61 | import           Control.Monad.Trans.Reader (ReaderT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:62:1: error:
    Could not find module ‘Control.Monad.Trans.State.Lazy’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
62 | import qualified Control.Monad.Trans.State.Lazy as Lazy (StateT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:63:1: error:
    Could not find module ‘Control.Monad.Trans.State.Strict’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
63 | import qualified Control.Monad.Trans.State.Strict as Strict (StateT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:64:1: error:
    Could not find module ‘Control.Monad.Trans.Writer.Lazy’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
64 | import qualified Control.Monad.Trans.Writer.Lazy as Lazy (WriterT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:65:1: error:
    Could not find module ‘Control.Monad.Trans.Writer.Strict’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
65 | import qualified Control.Monad.Trans.Writer.Strict as Strict (WriterT(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:66:1: error:
    Could not find module ‘Data.Functor.Classes’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
66 | import           Data.Functor.Classes
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:67:1: error:
    Could not find module ‘Data.Functor.Compose’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
67 | import           Data.Functor.Compose (Compose(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:68:1: error:
    Could not find module ‘Data.Functor.Constant’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
68 | import           Data.Functor.Constant (Constant(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:69:1: error:
    Could not find module ‘Data.Functor.Identity’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
69 | import           Data.Functor.Identity (Identity(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:70:1: error:
    Could not find module ‘Data.Functor.Product’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
70 | import           Data.Functor.Product (Product(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:71:1: error:
    Could not find module ‘Data.Functor.Reverse’
    There are files missing in the ‘transformers-0.5.2.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
71 | import           Data.Functor.Reverse (Reverse(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:72:1: error:
    Could not find module ‘Data.Functor.Sum’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
72 | import           Data.Functor.Sum ()
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:74:1: error:
    Could not find module ‘Control.Applicative’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
74 | import           Control.Applicative
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:75:1: error:
    Could not find module ‘Control.Monad’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
75 | import           Control.Monad (MonadPlus(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:76:1: error:
    Could not find module ‘Control.Monad.Fix’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
76 | import           Control.Monad.Fix (MonadFix(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:77:1: error:
    Could not find module ‘Data.Foldable’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
77 | import           Data.Foldable (Foldable(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:78:1: error:
    Could not find module ‘Data.Maybe’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
78 | import           Data.Maybe (fromMaybe)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:79:1: error:
    Could not find module ‘Data.Monoid’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
79 | import           Data.Monoid (Monoid(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:80:1: error:
    Could not find module ‘Data.Traversable’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
80 | import           Data.Traversable (Traversable(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:83:1: error:
    Could not find module ‘Control.Monad.Zip’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
83 | import           Control.Monad.Zip (MonadZip(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:87:1: error:
    Could not find module ‘Data.Bifunctor’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
87 | import           Data.Bifunctor (Bifunctor(..))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:91:1: error:
    Could not find module ‘Data.Data’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
91 | import           Data.Data (Data)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:92:1: error:
    Could not find module ‘Data.Typeable’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
92 | import           Data.Typeable
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Control/Monad/Trans/Instances.hs:95:1: error:
    Could not find module ‘GHC.Generics’
    There are files missing in the ‘base-4.10.0.0’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
95 | import           GHC.Generics
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-31447/transformers-compat-0.5.1.4'
cabal: Error: some packages failed to install:
StateVar-1.1.0.4-H2MMzWhEyt5Bk7Ng0WRNLx failed during the building phase. The
exception was:
ExitFailure 1
blaze-builder-0.4.0.2-9jN75sESh05JfWRLWr51Wl failed during the building phase.
The exception was:
ExitFailure 1
brick-0.19-CDfQnnzpr6W7uy2FsxJ10i depends on brick-0.19 which failed to
install.
contravariant-1.4-GLkdxedRfFlDA8zDtYtrgQ depends on contravariant-1.4 which
failed to install.
data-clist-0.1.2.0-GRWlwOEuIsQA9GA7i2yMwp failed during the building phase.
The exception was:
ExitFailure 1
dlist-0.8.0.3-8XCsMwviZunHg7vFDzvN5q failed during the building phase. The
exception was:
ExitFailure 1
herms-1.2.1.0-9cRgyzMdGq0ILOFGowGnbN depends on herms-1.2.1.0 which failed to
install.
microlens-0.4.8.1-HUhGDwljLwYJLac51xSk3I failed during the building phase. The
exception was:
ExitFailure 1
microlens-mtl-0.1.11.0-HZtiVpVdXuI1Wcea3wzDjp depends on
microlens-mtl-0.1.11.0 which failed to install.
microlens-th-0.4.1.1-LM8ksjmIHXtDJYY6zPs0W6 depends on microlens-th-0.4.1.1
which failed to install.
parallel-3.2.1.1-BDA8bigzIyf7HfOpmZ9w2K failed during the building phase. The
exception was:
ExitFailure 1
semigroups-0.18.3-9GRyDnRvoqO5HfdYR3Qz4o failed during the building phase. The
exception was:
ExitFailure 1
text-zipper-0.10.1-5anw58L5L9nGBx6ICflo2w failed during the building phase.
The exception was:
ExitFailure 1
transformers-compat-0.5.1.4-HHfl4KyHCH71PMx2CRgdve failed during the building
phase. The exception was:
ExitFailure 1
vty-5.15.1-4Es2l9VNfvyC2FLNsAICqc depends on vty-5.15.1 which failed to
install.
[joris@Marvin herms]$ cabal install -j >> output.txt
cabal: Error: some packages failed to install:
StateVar-1.1.0.4-H2MMzWhEyt5Bk7Ng0WRNLx failed during the building phase. The
exception was:
ExitFailure 1
blaze-builder-0.4.0.2-9jN75sESh05JfWRLWr51Wl failed during the building phase.
The exception was:
ExitFailure 1
brick-0.19-CDfQnnzpr6W7uy2FsxJ10i depends on brick-0.19 which failed to
install.
contravariant-1.4-GLkdxedRfFlDA8zDtYtrgQ depends on contravariant-1.4 which
failed to install.
data-clist-0.1.2.0-GRWlwOEuIsQA9GA7i2yMwp failed during the building phase.
The exception was:
ExitFailure 1
dlist-0.8.0.3-8XCsMwviZunHg7vFDzvN5q failed during the building phase. The
exception was:
ExitFailure 1
herms-1.2.1.0-9cRgyzMdGq0ILOFGowGnbN depends on herms-1.2.1.0 which failed to
install.
microlens-0.4.8.1-HUhGDwljLwYJLac51xSk3I failed during the building phase. The
exception was:
ExitFailure 1
microlens-mtl-0.1.11.0-HZtiVpVdXuI1Wcea3wzDjp depends on
microlens-mtl-0.1.11.0 which failed to install.
microlens-th-0.4.1.1-LM8ksjmIHXtDJYY6zPs0W6 depends on microlens-th-0.4.1.1
which failed to install.
parallel-3.2.1.1-BDA8bigzIyf7HfOpmZ9w2K failed during the building phase. The
exception was:
ExitFailure 1
semigroups-0.18.3-9GRyDnRvoqO5HfdYR3Qz4o failed during the building phase. The
exception was:
ExitFailure 1
text-zipper-0.10.1-5anw58L5L9nGBx6ICflo2w failed during the building phase.
The exception was:
ExitFailure 1
transformers-compat-0.5.1.4-HHfl4KyHCH71PMx2CRgdve failed during the building
phase. The exception was:
ExitFailure 1
vty-5.15.1-4Es2l9VNfvyC2FLNsAICqc depends on vty-5.15.1 which failed to
install.

Any suggestions? Would like to contribute to this project!

**Comment by gallais** [comment]
The travis build runs the build for 8.2.1 so it ought to work out of the box. In your log, this is concerning:

```
Could not find module ‘Prelude’
There are files missing in the ‘base-4.10.0.0’ package,
try running 'ghc-pkg check'.
```

What is the output of `ghc-pkg check`?

**Comment by Jorissss** [comment]
@gallais 
`ghc-pkg check`

Warning: haddock-interfaces: /usr/share/doc/haskell-hashtables/html/hashtables.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-hashtables/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-gtk2hs-buildtools/html/gtk2hs-buildtools.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-gtk2hs-buildtools/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-zlib/html/zlib.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-zlib/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-tar/html/tar.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-tar/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-stm/html/stm.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-stm/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-parsec/html/parsec.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-parsec/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-network/html/network.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-network/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-network-uri/html/network-uri.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-network-uri/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-http/html/HTTP.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-http/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-hackage-security/html/hackage-security.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-hackage-security/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-edit-distance/html/edit-distance.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-edit-distance/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-ed25519/html/ed25519.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-ed25519/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-echo/html/echo.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-echo/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-cryptohash-sha256/html/cryptohash-sha256.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-cryptohash-sha256/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-base64-bytestring/html/base64-bytestring.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-base64-bytestring/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-base16-bytestring/html/base16-bytestring.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-base16-bytestring/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-async/html/async.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-async/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-vector/html/vector.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-vector/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-hashable/html/hashable.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-hashable/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-tf-random/html/tf-random.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-tf-random/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-text/html/text.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-text/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-split/html/split.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-split/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-regex-posix/html/regex-posix.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-regex-posix/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-regex-base/html/regex-base.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-regex-base/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-random/html/random.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-random/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-quickcheck/html/QuickCheck.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-quickcheck/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-primitive/html/primitive.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-primitive/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/haskell-mtl/html/mtl.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/haskell-mtl/html doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/ghc-8.2.1/ghc.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/ghc-8.2.1 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/ghci-8.2.1/ghci.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/ghci-8.2.1 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/haskeline-0.7.4.0/haskeline.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/haskeline-0.7.4.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/terminfo-0.4.1.0/terminfo.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/terminfo-0.4.1.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/xhtml-3000.2.2/xhtml.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/xhtml-3000.2.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/ghc-compact-0.1.0.0/ghc-compact.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/ghc-compact-0.1.0.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/transformers-0.5.2.0/transformers.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/transformers-0.5.2.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/hoopl-3.10.2.2/hoopl.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/hoopl-3.10.2.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/template-haskell-2.12.0.0/template-haskell.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/template-haskell-2.12.0.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/ghc-boot-8.2.1/ghc-boot.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/ghc-boot-8.2.1 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/ghc-boot-th-8.2.1/ghc-boot-th.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/ghc-boot-th-8.2.1 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/Cabal-2.0.0.2/Cabal.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/Cabal-2.0.0.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/binary-0.8.5.1/binary.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/binary-0.8.5.1 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/pretty-1.1.3.3/pretty.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/pretty-1.1.3.3 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/hpc-0.6.0.3/hpc.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/hpc-0.6.0.3 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/process-1.6.1.0/process.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/process-1.6.1.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/directory-1.3.0.2/directory.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/directory-1.3.0.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/unix-2.7.2.2/unix.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/unix-2.7.2.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/time-1.8.0.2/time.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/time-1.8.0.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/containers-0.5.10.2/containers.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/containers-0.5.10.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/bytestring-0.10.8.2/bytestring.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/bytestring-0.10.8.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/deepseq-1.4.3.0/deepseq.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/deepseq-1.4.3.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/array-0.5.2.0/array.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/array-0.5.2.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/filepath-1.4.1.2/filepath.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/filepath-1.4.1.2 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/base-4.10.0.0/base.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/base-4.10.0.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/integer-gmp-1.0.1.0/integer-gmp.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/integer-gmp-1.0.1.0 doesn't exist or isn't a directory
Warning: haddock-interfaces: /usr/share/doc/ghc/html/libraries/ghc-prim-0.5.1.0/ghc-prim.haddock doesn't exist or isn't a file
Warning: haddock-html: /usr/share/doc/ghc/html/libraries/ghc-prim-0.5.1.0 doesn't exist or isn't a directory

**Comment by siddharthist** [comment]
@Jorissss Were you able to get it working? Please open another issue if not and I'll be happy to help troubleshoot. 

We have much more documentation in the README now, describing several different methods for installing Herms, so I'm closing this for now. Please reopen if the documentation needs changes or improvements!
